genshi/template/tests/eval.py

```diff
@@ -896,12 +896,12 @@ def __init__(self):
                 self.attr = 'foo'
         obj = Something()
         Suite("del obj.attr").execute({'obj': obj})
-        self.failIf(hasattr(obj, 'attr'))
+        self.assertFalse(hasattr(obj, 'attr'))
 
     def test_delitem(self):
         d = {'k': 'foo'}
         Suite("del d['k']").execute({'d': d})
-        self.failIf('k' in d, repr(d))
+        self.assertFalse('k' in d, repr(d))
```

Comment:
```suggestion
        self.assertNotIn('k', d)
```

This function emits a nice error message if it fails.